app/lib/specialist_publisher_wiring.rb

```diff
@@ -91,7 +91,7 @@
 
   define_factory(:countryside_stewardship_grant_builder) {
     SpecialistDocumentBuilder.new("countryside_stewardship_grant",
-    get(:validatable_document_factories).countryside_stewardship_grant_factory)
+      get(:validatable_document_factories).countryside_stewardship_grant_factory)
```

Comment:
I just made it consistent with the rest of the similar method invocations in the file. I can update them all, which one do you prefer?

1
```
  define_factory(:cma_case_builder) {
    SpecialistDocumentBuilder.new("cma_case",
      get(:validatable_document_factories).cma_case_factory
    )
  }
```

2
```
  define_factory(:cma_case_builder) {
    SpecialistDocumentBuilder.new(
      "cma_case",
      get(:validatable_document_factories).cma_case_factory
    )
  }
```

3
```
  define_factory(:cma_case_builder) {
    SpecialistDocumentBuilder.new("cma_case",
                                  get(:validatable_document_factories).cma_case_factory
    )
  }
```